substrate/frame/broker/src/utility_impls.rs

```diff
@@ -118,4 +117,9 @@ impl<T: Config> Pallet<T> {
 
 		Ok(Some((region_id, region)))
 	}
+
+	/// Returns the current block number of Relay Chain
+	pub(crate) fn now() -> RelayBlockNumberOf<T> {
```

Comment:
Maybe being a bit more explicit here is a good idea. Something like: `relay_height`?

Reply:
Yeah sounds like a good proposal to me.